action.yml

```diff
@@ -4,17 +4,21 @@ inputs:
   GITHUB_TOKEN:
     description: "GitHub token to interact with the repository."
     required: true
-  OPENAI_API_KEY:
-    description: "OpenAI API key for GPT."
+  INFERENCE_API_KEY:
```

Comment:
Consider using a more descriptive name for `INFERENCE_API_KEY`. Since it's not limited to OpenAI, a name like `LLM_API_KEY` might be more appropriate.